### PR TITLE
Create README, LICENSE and AUTHORS files

### PR DESCRIPTION
**Pull request by konoikon**

### CODE REVIEW

**Comment by pganssle** [comment]
@konoikon You have checked in a bunch of output files (all the ones under tmp/). Can you remove them? Might want to delete them from your local directory, too.

**Comment by pganssle** [comment]
Also, it seems like you are using the same branch as the first PR. You should pull the upstream branch into your master branch and create a new branch for each PR.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (pganssle) [inline LICENSE]:
```suggestion
   Copyright 2019 Grid Strategy Authors
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+a higher-order API for `GridStrategy` that also allows 
+for the proliferation of additional *figures* (e.g. 
+"if I have more than 10 axes to plot, split them 
+up as evenly as possible among `n / 10` different 
+figures"). This would be no harder to implement 
+in terms of the creation of such strategies, but 
+may be harder to work with since it would 
+necessarily spawn axes across multiple figures.
+
+Implementation
```
Comment (pganssle) [inline README.md]: We can drop this whole section.

**Comment by pganssle** [comment]
To fix the branch stuff, we're going to close this PR and start a new one from a different branch.

You can start with this:

```
git checkout master
git pull upstream master
```

If you don't have an `upstream`, you need to add one with:

```
git remote add upstream git@github.com:pganssle/grid-strategy.git
```

Then you want to do `git checkout -b add_meta_files` or whatever you want to call the branch. After you've done that, you can [cherry-pick your commit](http://think-like-a-git.net/sections/rebase-from-the-ground-up/cherry-picking-explained.html) into your new branch.

File: README.md (hunk 2 above)
Comment (konoikon) [reply README.md]: Should we drop the `higher dimensions` section as well?

Comment (pganssle) [reply README.md]: No I think the "higher dimensions" part is fine, we may want to refactor this README a bit, but this is a good start.

**Pull request closed without merging**